superset-frontend/src/common/components/Radio/index.tsx

```diff
@@ -21,20 +21,26 @@ import { Radio as BaseRadio } from 'src/common/components';
 
 const StyledRadio = styled(BaseRadio)`
   .ant-radio-inner {
-    width: 18px;
-    height: 18px;
+    top: -2px;
+    left: 2px;
+    width: 15px;
```

Comment:
I'm guessing you already thought of this, but my only teeeeeensy nitpick here is wondering if width/height look good at 16px, so we can use gridUnit * 4 here. Same with the 5px width/height below going down to possibly 4px (1 gridUnit). Hard for me to say just from looking at the code though ;)

Reply:
@evans I was checking the new design specification and actually the correct value is 16px! So now it's using gridUnit. I also changed the inner width and border to variate according to gridUnit because they should. Thanks for this review!